charts/trino/templates/configmap-access-control.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.accessControl }}{{- if eq .Values.accessControl.type "configmap" }}
```

Comment:
This is a good change, but we should still add this as a checksum annotation in the coordinator deployment if `.Values.accessControl.refreshPeriod` is not set.